packages/components/workbench/src/WorkbenchContent/WorkbenchContent.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { cx } from 'emotion';
+import { CommonProps, PropsWithHTMLElement } from '@contentful/f36-core';
+
+import { getWorkbenchContentStyles } from './WorkbenchContent.styles';
+
+export interface WorkbenchContentOwnProps extends CommonProps {
+  children: React.ReactNode;
+  type?: 'default' | 'text' | 'full';
+}
+
+export type WorkbenchContentProps = PropsWithHTMLElement<
+  WorkbenchContentOwnProps,
+  'main'
+>;
+
+export function WorkbenchContent({
+  children,
+  className,
+  type = 'default',
+  testId = 'cf-ui-workbench-content',
+  ...otherProps
+}: WorkbenchContentProps) {
+  const styles = getWorkbenchContentStyles(type);
+
+  return (
+    <main
```

Comment:
could you please incorporate latest changes from master? https://github.com/contentful/forma-36/pull/1268

Reply:
Do we still need it?
I'm extending the `<main>` attributes, so some one can do `<Workbench.Content id="some-id">` and it will work

We usually implement testId's but not ids
I tried to understand why this change was made in the PR you sent me, but there is no explanation

I volunteer to work on the Workbench v4 adoption in the product where this id is necessary and then I pass as a prop

wdyt?

Reply:
That is to be used with the `Skip to Content`, it jumps to that id provided there.